designs/l1_l4_security/l1_l4_security.md

```diff
@@ -0,0 +1,75 @@
+# L1-L4 security leveraging OpenStack security groups
+This document describes the notifications available for OpenStack security groups
+
+## Assignees
+Intel's OSC core team
+
+## Background
+OSC will support single page view to manage OpenStack security groups. User will be able to manage the OpenStack security groups and security group rules from OSC.
+
+## Constraints and Assumptions
+No addtional security groups are configured by user or admin directly on OpenStack.
```

Comment:
Does this mean that security groups should never be managed directly on a OpenStack environment for which OSC has a virtualization connector? What if the user does create, delete or update security groups on OpenStack? Will OSC ignore? overwrite? This assumption or constraint needs more clarity.